### DISCUSSION TITLE
Errors during Import-Export from 2.8.3 to 2.9.15

### DISCUSSION THREAD

**Discussion by lroi** [monarc-project/MonarcAppFO#491] [General]
**Describe the bug**
Hello, we have an import-export problem from 2.8.3 to 2.9.15. 
All the risk analysis exports from 2.8.3 are not imported in the same way and we have detected 3 different behaviors when importing from 2.9.15 :
- it works
- it looks as if it worked but the analysis is empty
- an error message saying that it doesn't work and the import that keeps going on forever in the background.

**To Reproduce**
1. On Monarc 2.8.3
2. Go to a risk analysis and click on "export..", "All", you download a .json file
3. On Monarc 2.9.15
4. Create an empty risk analysis with the same settings as the one you want to import (language, description, ..)
5. Click on "Import.." "by Duplication

**Expected behavior**
A proper export with everything in it and no errors.

**Comment by ruslanbaidan** [comment]
Hello,

As the first possible solution, we would like to ask you to try the import operation on the lates version of Monarc v2.10.1 (https://github.com/monarc-project/MonarcAppFO/releases/tag/v2.10.1). You can install it with use a VM or upgrade your version 2.9.15.

The other solution is to upgrade your Monarc instance 2.8.3 to 2.10.1, the instructions can be found on monarc.lu.
First you need to follow the steps here:
https://www.monarc.lu/news/2019/11/25/monarc-291-released/#updating
And after perform a regular update:
https://www.monarc.lu/news/2020/12/18/monarc-2101-released/
**Important** Before doing the upgrade it is very recommended to make a backup of the directory of monarc project as well as the database.

In order to be able to help you with the export issue from our side, we need to have your analyse exported file from the version 2.8.3.
If you would like to share it with us, the email address is info@cases.lu.
